WAF_wasm.cc

```diff
@@ -25,36 +25,40 @@ std::string printParams(QueryParams params) {
 }
 
 class ExampleRootContext : public RootContext {
-public:
-  explicit ExampleRootContext(uint32_t id, StringView root_id) : RootContext(id, root_id) {}
+ public:
+  explicit ExampleRootContext(uint32_t id, StringView root_id)
+      : RootContext(id, root_id) {}
 
   bool onConfigure(size_t config_size) override;
   bool onStart(size_t) override;
   Config getConfig() { return config_; }
 
-private:
+ private:
   struct Config config_;
 };
 
 class ExampleContext : public Context {
-public:
+ public:
   explicit ExampleContext(uint32_t id, RootContext* root) : Context(id, root) {}
 
   void onCreate() override;
-  FilterHeadersStatus onRequestHeaders(uint32_t headers, bool end_of_stream) override;
-  FilterDataStatus onRequestBody(size_t body_buffer_length, bool end_of_stream) override;
-  FilterHeadersStatus onResponseHeaders(uint32_t headers, bool end_of_stream) override;
+  FilterHeadersStatus onRequestHeaders(uint32_t headers,
```

Comment:
General comment (I just picked this line randomly): 
I've seen you've re-indented, which is good, you probably want to commit the `.indent.pro` file that would be used to re-indent the code. Possibly add a Makefile target, for any contributors who have `indent` installed to type `make indent` that would use the proper command line and/or local `.indent.pro`
Also, I think it's ok to set longer line, (`indent` is a very old tool which default line length to 80 characters, which might seem a bit restrictive. You probably want to change the line length default to a longer value (based on readability on a 1080p screen of two side-by-side files I'd say around 100 character length looks okay-ish.

Reply:
Thanks! I used `clang-format -style=Google` to reindent the files. Is that acceptable? Should I use `indent` instead?